modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/ballooneditor/plugin.js

```diff
@@ -69,12 +71,12 @@
 			CKEDITOR.ui.balloonPanel.DEFAULT_PANEL_PADDING = 14;
 
 			const getSelectionDirection = (selection) => {
-				let direction = CKEDITOR.SELECTION_TOP_TO_BOTTOM;
+				let direction = SELECTION_DIRECTION.TOP_TO_BOTTOM;
```

Comment:
Nice!  🎉 

We should avoid polluting CKEditor global. When seeing this LOC at the first time I tried to search on CKEDITOR docs and I didn't found anything.